apps/services/search-indexer/config/template-is.json

```diff
@@ -33,6 +33,17 @@
       },
       "analyzer": {
         "baseIcelandic": {
+          "type": "custom",
+          "tokenizer": "standard",
+          "filter": [
+            "lowercase",
+            "icelandicSynonym",
+            "icelandicStop",
+            "icelandicKeyword",
+            "icelandicStemmer"
+          ]
+        },
+        "compound": {
```

Comment:
Maybe chnage name of `compount` to `compoundIcelandic` to make it more clear in mapping that this is not a base elasticsearch analyzer

Reply:
Good catch, I completely agree 😉

This has been fixed